.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+fail_fast: false
+minimum_pre_commit_version: 4.0.0
+default_stages: [pre-commit]
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v6.0.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-case-conflict
+      - id: check-merge-conflict
+      - id: check-json
+      - id: check-toml
+      - id: check-yaml
+      - id: detect-private-key
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: "v0.13.0"
+    hooks:
+      - id: ruff-check
+        args: [--exit-non-zero-on-fix]
+      - id: ruff-format
+        args: [--exit-non-zero-on-fix]
```

Comment:
Isn't exit-non-zero the default? And/or, when pre-commit runs, if a delta is detected, doesn't the delta itself cause pre-commit to trip a failure anyways?